README.md

```diff
@@ -17,17 +17,22 @@ development stage (no stable release available).
 - Cross-platform (UNIX/Linux, Mac OS X, Windows)
 - Multilingual (application and library elements)
 - All-In-One: project management + schematic editor + board editor
-- Intuitive, modern and easy-to-use schematic/board editors
-- Very powerful library design
-- Uses XML as primary file format (allows version control of libraries and projects)
+- Intuitive, modern and easy-to-use graphical user interface
+- Very powerful library design with some innovative concepts
+- Uses human-readable file formats for both libraries and projects
 - Multi-PCB feature (different PCB variants of the same schematic)
-- Automatic netlist synchronisation between schematic and PCB (no manual export/import)
-- Up to 100 copper layers :-)
```

Comment:
In my opinion, yes. Of course we should document the layer limit somewhere, but I think the readme is the wrong place for such a obvious information.